src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -275,7 +295,31 @@ export default {
 				return `${loanUse}\n\n${this.description.substring(0, 120)}...`;
 			}
 			return 'For the borrower\'s privacy, this loan has been made anonymous.';
+		},
+		showUrgencyExp() {
+			return this.hasThreeDaysOrLessLeft && this.isUrgencyExpVersionShown;
 		}
 	},
+	created() {
```

Comment:
I added the experiment query here in created, since I had some issues with getting the experiment to work in prefetch in conjunction with calculating `hasThreeDaysOrLessLeft`

Reply:
I wonder if you could have chained promises to get `hasThreeDaysOrLessLeft` ready for a final segment that run the experiment assignment...

Reply:
Yep. too sticky for an exp!